build.py

```diff
@@ -297,16 +303,29 @@ def walk( project, configs, buildDir ) :
 
 		for pattern in configs[project].get( "manifest", [] ) :
 			for f in glob.glob( os.path.join( buildDir, pattern ) ) :
-				manifest.add( os.path.relpath( f, buildDir ) )
+				files.add( os.path.relpath( f, buildDir ) )
+
+		m = {
+			"name" : project,
+			"url" : configs[project]["url"],
+			"license" : "./" + project,
```

Comment:
We treat `license` as being optional elsewhere, should we be applying the same check here as we do for `credit`?https://github.com/GafferHQ/dependencies/blob/c521ac7f744b7644ce2c6dd91b5adf67e5661470/build.py#L215

Reply:
Indeed - fixed in aa8c696. Really we should enforce that all projects must have a license and then we can simplify this out. The culprit is GafferHQ/resources, which contains demo assets built by IE and a cow of mysterious origins :)